be/src/storage/rowset/bitmap_index_writer.cpp

```diff
@@ -40,14 +40,67 @@ namespace starrocks {
 
 namespace {
 
+class RoaringWrapper {
+    static const size_t roaring_get_size_fastpath_threshold = 1024;
+
+public:
+    explicit RoaringWrapper(rowid_t rid)
+            : _roaring(Roaring::bitmapOf(1, rid)),
+              _old_size(0),
+              _element_count(1),
+              _size_changed(true),
+              _enable_get_size_in_bytes(false){};
+
+    Roaring& roaring() { return _roaring; }
```

Comment:
```suggestion
    const Roaring& roaring() const { return _roaring; }
```

Reply:
this function is lator called `std::vector<Roaring*> bitmaps;`，it need a non const pointer